multipart/multipart_test.ts

```diff
@@ -0,0 +1,99 @@
+import { assertEqual, runTests, setFilter, test } from "../testing/mod.ts";
+import {
+  matchAfterPrefix,
+  MultipartReader,
+  MultipartWriter,
+  scanUntilBoundary
+} from "./multipart.ts";
+import { args, Buffer, cwd, open } from "deno";
+import * as path from "../fs/path.ts";
+
+const e = new TextEncoder();
+const d = new TextDecoder();
+const boundary = "--abcde";
+const dashBoundary = e.encode("--" + boundary);
+const nlDashBoundary = e.encode("\r\n--" + boundary);
+test(function testScanUntilBoundary1() {
+  const data = `--${boundary}`;
+  const [n, err] = scanUntilBoundary(
+    e.encode(data),
+    dashBoundary,
+    nlDashBoundary,
+    0,
+    "EOF"
+  );
+  assertEqual(n, 0);
+  assertEqual(err, "EOF");
+});
+test(function testScanUntilBoundary2() {
+  const data = `foo\r\n--${boundary}`;
+  const [n, err] = scanUntilBoundary(
+    e.encode(data),
+    dashBoundary,
+    nlDashBoundary,
+    0,
+    "EOF"
+  );
+  assertEqual(n, 3);
+  assertEqual(err, "EOF");
+});
+test(function testScanUntilBoundary4() {
```

Comment:
Rename this tests and others: multipartScanUntilBoundary4

Reply:
OK